static/common/js/registration.mjs

```diff
@@ -0,0 +1,57 @@
+export let getById = id => document.getElementById(id);
```

Comment:
Well those does not need to be exported, no? Also as functions within module, they can be `const`: 

`const let getById (...)`